packages/api/src/external/shared/patient-demographics.ts

```diff
@@ -0,0 +1,211 @@
+import { Patient, splitName, splitDob } from "@metriport/core/domain/patient";
+import { Address } from "@metriport/core/domain/address";
+import { Contact } from "@metriport/core/domain/contact";
+import { USState } from "@metriport/core/domain/geographic-locations";
+
+export const addressSeparator = "||";
+
+type LinkDemoDataAddress = {
+  line?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+};
+
+export type LinkDemoData = {
+  dob?: string;
+  gender?: string;
+  names: {
+    firstName: string;
+    lastName: string;
+  }[];
+  telephoneNumbers: string[];
+  emails: string[];
+  addresses: LinkDemoDataAddress[];
+  driversLicenses: {
+    value: string;
+    state: string;
+  }[];
+  ssns: string[];
+};
+
+export function scoreLink(
+  patientDemographics: LinkDemoData,
+  linkDemographics: LinkDemoData
+): boolean {
```

Comment:
if we are gonna use the epic algo, lets be explicit about that! I would convey that in a docstring at the minimum and possibly in the name of the fxn

Reply:
Great call. Done.

Reply:
nit: scoreLinkEpic. We decided on camelcase for naming convention

Reply:
Done.